### PR TITLE
Update CredentialsGitHubOptions

### PR DESCRIPTION
**Pull request by haldun**
Fixed the comments.

### CODE REVIEW

File: Sources/CredentialsGitHub/CredentialsGitHubOptions.swift [hunk 1]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  **/
 
-// MARK CredentialsFacebookOptions
-/// A list of options for authentication with Facebook.
+// MARK CredentialsGitHubOptions
+/// A list of options for authentication with Github.
```
Comment (ianpartridge) [inline Sources/CredentialsGitHub/CredentialsGitHubOptions.swift]: Should be `GitHub`.

**Review by ianpartridge — CHANGES_REQUESTED** [review]

**Comment by ianpartridge** [comment]
Thanks for your PR!  Have you submitted a [CLA](https://github.com/IBM-Swift/CLA)? If not, please do - it's a one-time thing which lets you contribute to all Swift@IBM projects.

**Comment by youming-lin** [comment]
@haldun Have you submitted the CLA yet? Please let us know here so we can decide on merging this PR for the next release.

**Comment by ianpartridge** [comment]
Already fixed.

**Pull request closed without merging**